addons/account/migrations/9.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,90 @@
+       &nbsp;&nbsp;Invoice date: ${object.date_invoice}<br />
+       % if object.origin:
+       &nbsp;&nbsp;Order reference: ${object.origin}<br />
+       % endif
+       % if object.user_id:
+       &nbsp;&nbsp;Your contact: <a href="mailto:${object.user_id.email or ''}?subject=Invoice%20${object.number}">${object.user_id.name}</a>
+       % endif
+    </p>
+
+    % if object.paypal_url:
+    <br/>
+    <p>It is also possible to directly pay with Paypal:</p>
+        <a style="margin-left: 120px;" href="${object.paypal_url}">
+            <img class="oe_edi_paypal_button" src="/account/static/src/img/btn_paynowcc_lg.gif"/>
+        </a>
+    % endif
+
+    <br/>
+    <p>If you have any question, do not hesitate to contact us.</p>
+    <p>Thank you for choosing ${object.company_id.name or 'us'}!</p>
+    <br/>
+    <br/>
+    <div style="width: 375px; margin: 0px; padding: 0px; background-color: #8E0000; border-top-left-radius: 5px 5px; border-top-right-radius: 5px 5px; background-repeat: repeat no-repeat;">
+        <h3 style="margin: 0px; padding: 2px 14px; font-size: 12px; color: #DDD;">
+            <strong style="text-transform:uppercase;">${object.company_id.name}</strong></h3>
+    </div>
+    <div style="width: 347px; margin: 0px; padding: 5px 14px; line-height: 16px; background-color: #F2F2F2;">
+        <span style="color: #222; margin-bottom: 5px; display: block; ">
+            ${object.company_id.partner_id.sudo().with_context(show_address=True, html_format=True).name_get()[0][1] | safe}
+        </span>
+        % if object.company_id.phone:
+            <div style="margin-top: 0px; margin-right: 0px; margin-bottom: 0px; margin-left: 0px; padding-top: 0px; padding-right: 0px; padding-bottom: 0px; padding-left: 0px; ">
+                Phone:&nbsp; ${object.company_id.phone}
+            </div>
+        % endif
+        % if object.company_id.website:
+            <div>
+                Web :&nbsp;<a href="${object.company_id.website}">${object.company_id.website}</a>
+            </div>
+        %endif
+        <p></p>
+    </div>
+</div>
+            ]]></field>
+      <field name="subject">${object.company_id.name} Invoice (Ref ${object.number or 'n/a'})</field>
+      <field name="email_from">${(object.user_id.email and '%s &lt;%s&gt;' % (object.user_id.name, object.user_id.email) or '')|safe}</field>
+    </record>
+    <record id="account_payment_term_immediate" model="account.payment.term">
+      <field name="note">Payment term: Immediate Payment</field>
+    </record>
+    <record id="sequence_reconcile_seq" model="ir.sequence">
+      <field eval="False" name="company_id"/>
+    </record>
+    <record id="account_payment_term_15days" model="account.payment.term">
+      <field name="note">Payment term: 15 Days</field>
+      <field name="line_ids" eval="[(0, 0, {'value': 'balance', 'value_amount': 0.0, 'sequence': 500, 'days': 15, 'option': 'day_after_invoice_date'})]"/>
```

Comment:
Remove this line, as there's no actual change (why is recognized by the script) and due to the addition operation, it's failing.

Reply:
but v8 doesn't have this line

Reply:
Right, I have seen v9 and v10, but this for v8 > v9. Any way, we have the reason why it appears, but it's still incorrect, because it's adding a second line, not replacing current one. Please replace it then with 2 commands: `[(5, ), (0, 0, ...`

Reply:
Done, let's see now.